main.go

```diff
@@ -188,6 +188,7 @@ func main() {
 		// error from os.RemoveAll is intentionally ignored, since there
 		// is no useful action we can do, and we don't need to confuse
 		// the user with an error message.
+
```

Comment:
This blank line is separating code from a comment that explains it. I think it doesn't belong here.